src/rubocop/helpers/doc-cache.js

```diff
@@ -20,6 +20,10 @@ function takeWhile(source, predicate) {
   return result
 }
 
+function formatDoc(text) {
+  return `<pre>${text.replace(/\[source,ruby\]|----/gi, '')}</pre>`
```

Comment:
Just a note that this isn't Markdown, although I'm not sure what markup language it is 🤷‍♂.

Reply:
Yep, it is not markdown at all, I don't know what is it too, but the ruby comments with `# `are formatted as markdown.